website/public/js/controllers/groupsCtrl.js

```diff
@@ -249,6 +260,7 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
     }
 
     $scope.postChat = function(group, message){
+      $scope.preview = false;
```

Comment:
If you move the preview logic into a directive as mentioned above, you can probably add some kind of listener that resets this (can't think of a specific solution right now... sleepy... but I think it's there).